schemas/common_definitions.json

```diff
@@ -406,5 +406,14 @@
   "interviewer_only": {
     "description": "Marks the page as for the interviewer only. Only applies for questionnaires that will be completed by an interviewer.",
     "type": "boolean"
+  },
+  "introduction_guidance": {
+    "description": "Introduction guidance",
+    "type": "object",
+    "properties": {
+      "contents": {
+        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/contents"
+      }
+    }
   }
```

Comment:
We should name this just "guidance", then use it for both intro and question page. Atm, we have another `question_guidance` which is a duplicate.

Reply:
So only `guidance` now, thanks!